src/session.rs

```diff
@@ -31,18 +35,45 @@ impl SessionWrapper {
             .build()
             .await
             .map_err(err_to_napi)?;
-        Ok(SessionWrapper {
-            internal_session: sb,
-        })
+        Ok(SessionWrapper { internal: sb })
     }
 
     #[napi]
     pub async fn query_unpaged_no_values(&self, query: String) -> napi::Result<QueryResultWrapper> {
         let query = self
-            .internal_session
+            .internal
             .query_unpaged(query, &[])
             .await
             .map_err(err_to_napi)?;
         Ok(QueryResultWrapper::from_query(query))
     }
+
+    #[napi]
+    pub async fn prepare_statement(
+        &self,
+        statement: String,
+    ) -> napi::Result<PreparedStatementWrapper> {
+        Ok(PreparedStatementWrapper {
+            query: self
+                .internal
+                .prepare(statement)
+                .await
+                .map_err(err_to_napi)?,
+        })
+    }
+
+    #[napi]
+    pub async fn query_simple(
```

Comment:
Please provide docstrings to these functions, as they are crucial for the new driver.